kipet/examples/Ex_15_estimation_mult_exp_unwanted_G.py

```diff
@@ -0,0 +1,173 @@
+#  _________________________________________________________________________
+#
+#  Kipet: Kinetic parameter estimation toolkit
+#  Copyright (c) 2016 Eli Lilly.
+#  _________________________________________________________________________
+
+# Sample Problem 
+# Estimation with unknow variancesof spectral data using pyomo discretization 
+#
+#		\frac{dZ_a}{dt} = -k_1*Z_a	                Z_a(0) = 1
+#		\frac{dZ_b}{dt} = k_1*Z_a - k_2*Z_b		Z_b(0) = 0
+#               \frac{dZ_c}{dt} = k_2*Z_b	                Z_c(0) = 0
+#               C_k(t_i) = Z_k(t_i) + w(t_i)    for all t_i in measurement points
+#               D_{i,j} = \sum_{k=0}^{Nc}C_k(t_i)S(l_j) + \xi_{i,j} for all t_i, for all l_j 
+#       Initial concentration 
+
+from __future__ import print_function
+from kipet.library.TemplateBuilder import *
```

Comment:
I think we should move away from using global imports. It is generally frowned upon.